packages/hdwallet-tallyho/src/tallyho.ts

```diff
@@ -91,6 +91,7 @@ export class TallyHoHDWallet implements core.HDWallet, core.ETHWallet {
   readonly _supportsAvalanche = false;
   readonly _supportsOptimism = false;
   readonly _supportsPolygon = true;
+  readonly _supportsGnosis = true;
```

Comment:
not supported out of the box
```suggestion
  readonly _supportsGnosis = false;
```

Reply:
Actually, fresh from a few hours ago, should be safe to add

> You can now use any EVM network listed on https://chainlist.org/ inside Taho! Including zksync, fantom, canto, celo, gnosis, moonbeam, etc. 🎉 

<img width="403" alt="image" src="https://github.com/shapeshift/hdwallet/assets/17035424/462706f4-99d8-4965-bb31-d832f9c74373">

But yeah either or, we don't really support this wallet

Reply:
Also this would require turning on ethSwithChain support for Tally, which doesn't matter since we've removed it as an option in the app 👋🏽